src/features/AddModel/AddModelDialog/AddModelDialog.tsx

```
@@ -59,6 +60,18 @@ export const AddModelDialog = ({
   const [fileSize, setFileSize] = useState(0);
   const [rawFile, setrawFile] = useState<File>();
 
+  useEffect(() => {
+    if (Object.keys(errors).length === 0 && submitting) {
+      finishSubmit();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [errors, submitting]);
```

There are some situations where exhaustive-deps error could be ignored. However, this does not look like the case. Suggest reading [this ](https://legacy.reactjs.org/docs/hooks-faq.html#is-it-safe-to-omit-functions-from-the-list-of-dependencies) article and try to comply with it in the future.